scripts/util/release.rb

```diff
@@ -32,7 +32,7 @@ def initialize(hash)
       @date = hash.fetch("date")
       @name = hash.fetch("name")
       @version = Util::Version.new(@name)
-      @whats_next = hash.fetch("whats_next")
+      @whats_next = hash.fetch("whats_next", [])
```

Comment:
This was previously broken?

Reply:
It allows omitting this key from the releases. Historically these have been inaccurate so I've stopped publishing them. Previously this required setting `whats_next: []` in each release, but this defaults it.